daemon/src/identify/dialer.rs

```diff
@@ -158,3 +158,17 @@ impl Actor {
         }
     }
 }
+
+const WIRE_VERSION_LABEL: &str = "wire_version";
+const DAEMON_VERSION_LABEL: &str = "daemon_version";
+const ENVIRONMENT_LABEL: &str = "environment";
+
+static NUM_LIBP2P_CONNECTIONS_GAUGE: conquer_once::Lazy<prometheus::IntGaugeVec> =
```

Comment:
Ah, here it is. I guess the first commit does not build? 

This metric says `number of active libp2p connections` but it looks like you are tracking the number of received `IdentifyMsgReceived` messages.